src/main/kotlin/com/cognifide/gradle/aem/pkg/deploy/PackageError.kt

```diff
@@ -0,0 +1,23 @@
+package com.cognifide.gradle.aem.pkg.deploy
+
+enum class PackageError(val className: String) {
+    CONSTRAINT_VIOLATION_EXCEPTION("javax.jcr.nodetype.ConstraintViolationException"),
```

Comment:
Is there any need to have enum? Why not just having list of strings in AemConfig?